database/media_query.py

```diff
@@ -98,3 +98,18 @@ def _query_async(self):
         event_key = ndb.Key(Event, self._query_args[0])
         medias = yield Media.query(Media.references == event_key).fetch_async()
         raise ndb.Return(medias)
+
+
+class TeamTagMediasQuery(DatabaseQuery):
```

Comment:
Cache clearing

Reply:
This query still needs cache clearing ([in here](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/database/get_affected_queries.py#L107))